components/style/properties/gecko.mako.rs

```diff
@@ -2988,18 +2989,27 @@ ${define_ffi_struct_accessor(style_struct)}
 % endfor
 
 lazy_static! {
-    static ref EMPTY_VARIABLES_STRUCT: nsStyleVariables = {
-        unsafe {
-            let mut variables: nsStyleVariables = unsafe { zeroed() };
-            Gecko_Construct_nsStyleVariables(&mut variables);
-            variables
-        }
-    };
+    static ref EMPTY_VARIABLES_STRUCT: Mutex<Option<nsStyleVariables>> = Mutex::new(None);
 }
 
 #[no_mangle]
 #[allow(non_snake_case)]
 pub unsafe extern "C" fn Servo_GetStyleVariables(_cv: ServoComputedValuesBorrowedOrNull)
```

Comment:
It used to be, but we moved it over here when we had some linking issues in non-MOZ_STYLO builds.